### PR TITLE
Updated Brazilian Portuguese translations

### PR DESCRIPTION
**Pull request by leonardoferreiradg**
I think now that PT-BR Locale was updated and is ready for production.

I reviewed the grammar, vocabulary and finish the phrases without a translation. We've a fully translation for pt-BR now (I'm brazilian and I really need it for my community rsrs).

### CODE REVIEW

**Comment by rogix** [comment]
Hi, @leonardoferreiradg Great to have another Brazilian working on this :). I opened this PR a few days ago to solve the same issue. Would you mind consider reviewing it instead? #16968

**Comment by leonardoferreiradg** [comment]
@daniellockyer done! Please, help me to join this awesome company. I'm a Product Designer working from Brazil 😄 

@rogix totally, man! I have a subscription product based in Ghost, so, it will be very helpfull for me and other creators. Check it out leonardoferreira.design

Thanks again, guys! Let's go foward!!

File: ghost/i18n/locales/pt-BR/ghost.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
     "All the best!": "Tudo de bom!",
+    "Complete signup for {{siteTitle}}!": "Complete sua criação de conta em {{siteTitle}}!",
```
Comment (daniellockyer) [inline ghost/i18n/locales/pt-BR/ghost.json:3]: This block introduces duplicate keys

**Review by rogix — APPROVED** [review]
Sounds good to me :)

**Comment by daniellockyer** [comment]
Thanks @leonardoferreiradg! 🙂

**Pull request merged**